src/dfx/src/lib/installers/assets/mod.rs

```diff
@@ -8,13 +8,13 @@ use ic_agent::Agent;
 use std::time::Duration;
 
 #[context("Failed to store assets in canister '{}'.", info.get_name())]
-pub async fn post_install_store_assets(
+pub async fn post_install_store_frontend(
```

Comment:
All changes in this file should be reverted
```suggestion
pub async fn post_install_store_assets(
```